### DISCUSSION TITLE
remove "doesn't work for pytorch" from `SimBA` tests

### DISCUSSION THREAD

**Discussion by Embeddave** [Trusted-AI/adversarial-robustness-toolbox#1409] [Ideas]
**Is your feature request related to a problem? Please describe.**
When troubleshooting the `SimBA` attack I found several places in the tests that state it does not work for `torch`:

https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/85ca10ff6b696dbaa66dcf179d0493b12ff594bb/tests/attacks/test_simba.py#L42

https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/85ca10ff6b696dbaa66dcf179d0493b12ff594bb/tests/attacks/test_simba.py#L97

(which is particularly surprising since it was originally written in `torch`)

**Describe the solution you'd like**
Remove that language from comments in tests or please clarify if there's something I'm not understanding.
After troubleshooting I was able to run the attacks with a `torch` model.
It does seem to be working for me, although I haven't done extensive runs yet.

**Comment by beat-buesser** [comment]
Moved back to issue #1410, Please apologise the confusion.
